### PR TITLE
Fix codacy warnings

### PR DESCRIPTION
**Pull request by lodamar**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -74,11 +71,11 @@ Community leaders will follow these Community Impact Guidelines in determining t
 ## Attribution
 
 This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 2.0,
-available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+available at <https://www.contributor-covenant.org/version/2/0/code_of_conduct.html>.
 
 Community Impact Guidelines were inspired by [Mozilla's code of conduct enforcement ladder](https://github.com/mozilla/diversity).
 
-[homepage]: https://www.contributor-covenant.org
```
Comment (massimosiani) [inline CODE_OF_CONDUCT.md]: why moved below?

File: src/test/scala/it/bitrock/dvs/streams/TestValues.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ object TestValues {
   final val ParamsAirport4 = AirportParams("NAP", "Napoli Capodichino", "IT")
   final val ParamsAirport5 = AirportParams("NCE", "Nice", "FR")
   final val ParamsAirport6 = AirportParams("AEI", "Algeciras", "ES")
-  final val ParamsAirport7 = AirportParams("BVG", "Berlevåg", "NO")
```
Comment (massimosiani) [inline src/test/scala/it/bitrock/dvs/streams/TestValues.scala]: what's the corresponding rule?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (lodamar) [reply CODE_OF_CONDUCT.md]: Codacy suggests to have these at the bottom of the file, it should be the standard. 
The place it's used hasn't changed

File: src/test/scala/it/bitrock/dvs/streams/TestValues.scala (hunk 2 above)
Comment (lodamar) [reply src/test/scala/it/bitrock/dvs/streams/TestValues.scala]: Forbid non ASCII characters

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (massimosiani) [reply CODE_OF_CONDUCT.md]: did you check the resulting output? I think we should escape the brackets instead of blindly obey to suggestions :)

Comment (massimosiani) [reply CODE_OF_CONDUCT.md]: my fault.

**Pull request merged**